### PR TITLE
curvefs/tool: optimize output

### LINKED ISSUE #794: curvefs_tool: curvefs_tool version  error and curvefs_tool status command does not exist
**Describe the bug (描述bug)**

**To Reproduce (复现方法)**
root@pubbeta2-curve4:/usr/local/curvefs/etcd# curvefs_tool version
unknown

root@pubbeta2-curve4:/usr/local/curvefs/etcd# curvefs_tool status
Usage: curvefs_tool [Command] [OPTIONS...]
COMMANDS:
version: show the version of cluster
build-topology: build cluster topology based on topo.json
umount-fs: umount curvefs from local and cluster
usage-metadata: show the metadata usage of cluster
status-mds: show the status of mds

**Expected behavior (期望行为)**

**Versions (各种版本)**
OS:
Compiler:
curve-mds:
curve-chunkserver:
curve-snapshotcloneserver:
curve-sdk:
nebd:
curve-nbd:

**Additional context/screenshots (更多上下文/截图)**

### PR DESCRIPTION
**Pull request by Cyber-SiKu**
### What problem does this PR solve?

Issue Number: close #794 #793 #792  

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: curvefs/src/mds/mds.h [hunk 1]
```diff
@@ -31,6 +31,7 @@
 #include "curvefs/src/mds/chunkid_allocator.h"
 #include "curvefs/src/mds/fs_manager.h"
 #include "src/common/configuration.h"
+#include "src/common/curve_version.h"
```
Comment (wu-hanqing) [inline curvefs/src/mds/mds.h:34]: put this include to .cpp file

File: curvefs/src/metaserver/metaserver.h [hunk 2]
```diff
@@ -38,6 +38,7 @@
 #include "curvefs/src/metaserver/inflight_throttle.h"
 #include "curvefs/src/metaserver/metaserver_service.h"
 #include "src/common/configuration.h"
+#include "src/common/curve_version.h"
```
Comment (wu-hanqing) [inline curvefs/src/metaserver/metaserver.h:41]: ditto

File: curvefs/src/tools/README.md [hunk 3]
```diff
@@ -0,0 +1,501 @@
...
+
+A tool for CurveFS.
+
+Usage:
+
+```shell
+curvefs_tool [Command] [OPTIONS...]
+```
+
+When you are not sure how to use a command, --exapmle can give you an example of use:
```
Comment (wu-hanqing) [inline curvefs/src/tools/README.md:11]: typo: --example

of usage ?

File: curvefs/src/tools/BUILD [hunk 4]
```diff
@@ -66,6 +67,7 @@ cc_library(
         "buildtopo/curvefs_build_topology_tool.h",
         "usage/curvefs_metadata_usage_tool.h",
         "usage/curvefs_space_base_tool.h",
+        "status/curvefs_status.h",
```
Comment (wu-hanqing) [inline curvefs/src/tools/BUILD:70]: you didn't get the idea of `glob`

```
glob([
   "*.h",
   "dir1/*.h",
   "dir2/*.h",
])
```

is enough

File: curvefs/src/tools/BUILD [hunk 5]
```diff
@@ -39,6 +39,7 @@ cc_library(
         "version/curvefs_version_tool.cpp",
         "usage/curvefs_metadata_usage_tool.cpp",
         "usage/curvefs_space_base_tool.cpp",
+        "status/curvefs_status.cpp",
```
Comment (wu-hanqing) [inline curvefs/src/tools/BUILD:42]: ditto

File: curvefs/src/tools/README.md [hunk 6]
```diff
@@ -0,0 +1,501 @@
...
+Usage:
+
+```shell
+curvefs_tool build-topology
+```
+
+Output:
+
+```shell
+
```
Comment (wu-hanqing) [inline curvefs/src/tools/README.md:253]: empty ?

File: curvefs/src/tools/README.md [hunk 7]
```diff
@@ -0,0 +1,501 @@
...
+```shell
+curvefs_tool delete-fs -fsName=/test -confirm
+```
+
+Output:
+
+```shell
+1. do you really want to delete fs (/test) :[Ny]y
+2. do you really want to delete fs (/test) :[Ny]y
+3. do you really want to delete fs (/test) :[Ny]y
```
Comment (wu-hanqing) [inline curvefs/src/tools/README.md:324]: warning three times?

File: curvefs/util/build.sh [hunk 8]
```diff
@@ -3,6 +3,25 @@
 # Copyright (C) 2021 Jingli Chen (Wine93), NetEase Inc.
 
 ############################  GLOBAL VARIABLES
+#get tag version
+tag_version=`git status | grep -w "HEAD detached at" | awk '{print $NF}' | awk -F"v" '{print $2}'`
+if [ -z ${tag_version} ]
+then
+    echo "not found version info, set version to 9.9.9"
+    tag_version=9.9.9
```
Comment (wu-hanqing) [inline curvefs/util/build.sh:11]: do not use "9.9.9", use another mark or only commit id

File: curvefs/util/build.sh [hunk 9]
```diff
@@ -3,6 +3,25 @@
...
+# get git commit version
+commit_id=`git show --abbrev-commit HEAD|head -n 1|awk '{print $2}'`
+if [ "$1" = "debug" ]
+then
+    debug="+debug"
+else
+    debug=""
+fi
+
+curvefs_version=${tag_version}+${commit_id}${debug}
```
Comment (wu-hanqing) [inline curvefs/util/build.sh:23]: encapsulate into function

File: src/common/curve_version.cpp [hunk 10]
```diff
@@ -40,11 +40,29 @@ std::string CurveVersion() {
...
     version.expose_as("curve", "version");
     version.set_value(CurveVersion());
 }
 
+void ExposeCurvefsVersion() {
+    static bvar::Status<std::string> version;
+    version.expose_as("curvefs", "version");
+    version.set_value(CurvefsVersion());
+}
+
```
Comment (wu-hanqing) [inline src/common/curve_version.cpp:66]: why add another one?

File: curvefs/src/mds/mds.h (hunk 1 above)
Comment (Cyber-SiKu) [reply curvefs/src/mds/mds.h:34]: fix

File: curvefs/src/metaserver/metaserver.h (hunk 2 above)
Comment (Cyber-SiKu) [reply curvefs/src/metaserver/metaserver.h:41]: > ditto

fxi

File: curvefs/src/tools/README.md (hunk 3 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:11]: fix

File: curvefs/src/tools/BUILD (hunk 4 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/BUILD:70]: fix

File: curvefs/src/tools/BUILD (hunk 5 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/BUILD:42]: > ditto

fix

File: curvefs/src/tools/README.md (hunk 6 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:253]: yep

File: curvefs/src/tools/README.md (hunk 7 above)
Comment (Cyber-SiKu) [reply curvefs/src/tools/README.md:324]: > warning three times?

will fix in next

File: curvefs/util/build.sh (hunk 8 above)
Comment (Cyber-SiKu) [reply curvefs/util/build.sh:11]: fix

File: src/common/curve_version.cpp (hunk 10 above)
Comment (Cyber-SiKu) [reply src/common/curve_version.cpp:66]: > 
Keep the difference from curve to distinguish

**Comment by Cyber-SiKu** [comment]
recheck

**Pull request merged**